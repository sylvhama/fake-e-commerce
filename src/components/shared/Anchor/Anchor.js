import styled from 'styled-components';

export default styled.a`
  color: ${props => props.theme.color.primary};
  text-decoration: none;

  &:hover {
    text-decoration: underline;
  }
`;
