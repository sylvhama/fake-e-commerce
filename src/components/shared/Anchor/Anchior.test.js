import React from 'react';
import 'jest-styled-components';

import { renderWithTheme } from '../../../tests/helpers';

import Anchor from './Anchor';

test('Anchor renders correctly', () => {
  const rendered = renderWithTheme(<Anchor>Anchor</Anchor>).toJSON();
  expect(rendered).toMatchSnapshot();
});
