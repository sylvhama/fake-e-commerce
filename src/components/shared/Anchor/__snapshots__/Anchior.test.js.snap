// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Anchor renders correctly 1`] = `
.c0 {
  color: #ea352d;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<a
  className="c0"
>
  Anchor
</a>
`;
